authorization-service/serverless.ts

```diff
@@ -0,0 +1,57 @@
+import {Serverless} from "serverless/plugins/aws/provider/awsProvider";
+
+const serverlessConfiguration: Serverless = {
+    service: {
+        name: 'authorization-service',
+    },
+    frameworkVersion: '2',
+    custom: {
+        webpack: {
+            webpackConfig: './webpack.config.js',
+            includeModules: true
+        }
+    },
+    plugins: [
+        'serverless-webpack',
+        'serverless-dotenv-plugin'
+    ],
+    provider: {
+        name: 'aws',
+        runtime: 'nodejs12.x',
+        iamRoleStatements: [
```

Comment:
I don't think that we need access to all this resources from authorization lambda